operator/internal/controller/redpanda/redpanda_controller_test.go

```diff
@@ -819,6 +819,11 @@ func (s *RedpandaControllerSuite) minimalRP(useFlux bool) *redpandav1alpha2.Redp
 		Spec: redpandav1alpha2.RedpandaSpec{
 			ChartRef: redpandav1alpha2.ChartRef{
 				UseFlux: ptr.To(useFlux),
+				Upgrade: &redpandav1alpha2.HelmUpgrade{
+					Remediation: &v2beta2.UpgradeRemediation{
+						Retries: 5,
```

Comment:
Please add a comment about why this has been added :)

Reply:
Done